### PR TITLE
Adds a default PassThroughRule for tag `<sup>`

### LINKED ISSUE #615: <sup> should be a common rule by default
Hi,

I think `<sup>` should be a common rule by default

### PR DESCRIPTION
**Pull request by everton-rosario**
This PR:

* [x] Adds PassThroughRule for tag `<sup>`

Fixes #615

### CODE REVIEW

File: rules-configuration.json [hunk 1]
```diff
@@ -27,6 +27,9 @@
         "class": "ParagraphRule",
         "selector": "p"
     }, {
+        "class": "PassThroughRule",
+        "selector": "sup"
```
Comment (vkama) [inline rules-configuration.json]: Is there a reason that `<sub>` was not added too?

**Comment by timjacobi** [comment]
@everton-rosario What work is left here to get this merged?

**Comment by timjacobi** [comment]
Actually I'm not sure if I agree with this PR. I feel like we should rather define a new rule that converts a character wrapped in a `<sub>` tag to its superscript equivalent. Otherwise the widely used m² would be converted to m2 for example. I think that would make people unhappy.

**Comment by paulschreiber** [comment]
@timjacobi can you propose a patch that does what you want?

**Comment by jerclarke** [comment]
Someone at facebook, please point out how absurd it is that these simple HTML tags don't work in IA. 

It's sad that we have to consider these options.

**Comment by everton-rosario** [comment]
Somehow this got lost in between releases.

I'll update the PR with the proper tags added.

**Comment by everton-rosario** [comment]
We do render it not so gracefully into the UI of Instant Articles, though we do support the `<sub>`/`<sup>` tags.

I'll possibly change this PR to properly have the tags and the proper rules, so we don't need to bypass them, rendering it properly as `sup` and `sub` tags.

**Pull request closed without merging**